src/amici_interface_matlab.cpp

```diff
@@ -160,6 +138,7 @@ UserData *userDataFromMatlabCall(const mxArray *prhs[], int nrhs,
         readOptionScalarInt(stldet, booleantype);
         readOptionData(qpositivex);
         readOptionScalarInt(sensi, AMICI_sensi_order);
+        // this needs to be set before we set parameters to obtain properly unscaled parameters
```

Comment:
Are the unscaled parameters updated when pscale is changed? I think they should be, otherwise this begs for trouble. But I agree that we should set pscale first.

Reply:
Now they are.